account_move_batch_validate/wizard/move_marker_view.xml

```diff
@@ -2,58 +2,22 @@
 <openerp>
     <data>
 
-        <record id="view_account_move_marker" model="ir.ui.view">
-            <field name="name">Mark Jornal Items for Batch Posting</field>
-            <field name="model">account.move.marker</field>
+        <record id="validate_account_move_view" model="ir.ui.view">
+            <field name="name">Post Journal Entries</field>
+            <field name="model">validate.account.move</field>
+            <field name="inherit_id" ref="account.validate_account_move_view" />
             <field name="arch" type="xml">
-                <form string="Report Options" version="7.0">
-                    <label string=""/>  <!-- binding for inherited views -->
-                    <group col="4">
-                        <field name="chart_account_id" widget='selection' on_change="onchange_chart_id(chart_account_id, context)"/>
-                        <field name="company_id" invisible="1"/>
-                        <field name="fiscalyear_id" domain="[('company_id','=',company_id)]"/>
-                        <field name="action"/>
+                <xpath expr="//group" position="before">
+                    <group>
+                        <field name="asynchronous" />
+                        <field name="action" attrs="{'invisible': [('asynchronous', '=', False)]}"/>
                     </group>
```

Comment:
Can you try without creating a new group for asynchronous and action? The look should be better.